modules_basic/avatar/image.js

```diff
@@ -74,6 +76,24 @@ exports.create = function (api) {
       )
     },
 
+    avatar_image_src: function (author) {
+      var src = ready && avatars[author] ? api.blob_url(avatars[author].image) : genSrc(author)
+
```

Comment:
yeah that's what I thought. 
The interface still seems to work despite this, so I'll jsut drop this